### ISSUE TITLE
Cannot Reproduce RE Results of XFUND-zh

### ISSUE THREAD

**Issue by davidz123** [microsoft/unilm#427]
Hi，I am trying to Reproduce Results of LayoutXLM on XFUND-zh(RE task).
But The result is quite different, my f1-score is 0.073(vs 0.707 as you posed).
I am wondering why this case happen, I did some changes of code and model:
     1. I downloaded layoutxlm-base model offline cause My work-env do not support Internet connection.
         And Waring shows some params(in extractor, and not BN params) is not Initialized
     2. I just test layoutxlm model with params from huggingface directly (Without Finetune) , is that the reason why the F1-score such low?

**Comment by wolfshow** [comment]
@davidz123 of coursce you need to fine-tune the model, otherwise you only got the pre-trained model which has never seen the xfund labels.
